### PR TITLE
New petabjl syntax

### PR DESCRIPTION
**Pull request by PaulJonasJost**
new example taken from [here](https://sebapersson.github.io/PEtab.jl/stable/Boehm/)

### CODE REVIEW

File: doc/example/conversion_reaction/PEtabJl_module.jl [hunk 1]
```diff
@@ -5,15 +5,15 @@ using Sundials
 using PEtab
 
 pathYaml = "/Users/pauljonasjost/Documents/GitHub_Folders/pyPESTO/test/julia/../../doc/example/conversion_reaction/conversion_reaction.yaml"
-petabModel = readPEtabModel(pathYaml, verbose=true)
+petabModel = PEtabModel(pathYaml, verbose=true)
 
 # A full list of options for createPEtabODEProblem can be found at https://sebapersson.github.io/PEtab.jl/dev/API_choosen/#PEtab.setupPEtabODEProblem
```
Comment (sebapersson) [inline doc/example/conversion_reaction/PEtabJl_module.jl:10]: Change to PEtabODEProblem in the coment (createPEtabODEProblem no longer exists)

**Review by sebapersson — CHANGES_REQUESTED** [review]
Very minor comments. I will also try to fix the API doc in PEtab.jl to make it easy for anyone to see all the options.

**Review by Doresic — APPROVED** [review]
Looks good, a better naming scheme than before.

File: doc/example/conversion_reaction/PEtabJl_module.jl (hunk 1 above)
Comment (PaulJonasJost) [inline doc/example/conversion_reaction/PEtabJl_module.jl:10]:
```suggestion
# A full list of options for PEtabODEProblem can be found at https://sebapersson.github.io/PEtab.jl/stable/API_choosen/
```

**Pull request merged**